build.sh

```diff
@@ -5,7 +5,7 @@ if [ -z "$IMAGE" ]; then
 fi
 
 if [ -z "$TAG" ]; then
-    TAG="0.2.0"
+    TAG="v2.1.0"
```

Comment:
I think if this is meant for the final 2.1.0 release, it should be `2.1.0`, for consistency with our past releases.